chart/templates/mayastor/rbac/rbac.yaml

```diff
@@ -20,10 +20,14 @@ rules:
 - apiGroups: ["apiextensions.k8s.io"]
   resources: ["customresourcedefinitions"]
   verbs: ["create", "get", "update", "list"]
+  # must read mayastorpools info
```

Comment:
maybe extend comment to say that it is to handle upgrades from v1

Reply:
Done.